### ISSUE TITLE
clarify distinction between "worldwide" and "remote"

### ISSUE THREAD

**Issue by waldyrious** [fossjobs/fossjobs#15]
If "worldwide" means "any location other than the ones listed above", then maybe an additional "remote" category could be created. Otherwise, the name "worldwide" should be clarified.

**Comment by waldyrious** [comment]
Ah, looking at [the code](https://github.com/fossjobs/fossjobs/blob/master/public/_templates/bootstrap/_translations/translations_en.ini#L43) clarifies this:
> `other_countries = "Worldwide"`

So my suggestion is to add a separate "remote" category.

**Comment by moba** [comment]
Looking at past submissions, and how people have been using it, and the meaning of "worldwide", I would say there is no difference between "Worldwide" as job *location*, and "remote". What difference do you see?

If the job requires you to work in city X or country X, it's not worldwide, and not "remote". If the job allows you to work from anywhere, it is "worldwide", and "remote".

I understand there _are_ differences, but I don't see how they have any relevance for us given the low volume of FOSS jobs and the goal of making it simple for everyone. If we introduce a new option, who will go back and update all submissions, and who will in the future be able to make the decision based on the limited information in calls? Does it matter so much in practice? Is the imagined difference between "worldwide" and "remote" actually going to make a difference in the sense that you can get in touch with who offered the job to find out the details?

**Comment by waldyrious** [comment]
I see. It wasn't clear to me that the list of locations in the sidebar were not hardcoded. The fact that the other division (categories) is indeed a hardcoded list probably aided to this perception. When I looked at the code, I concluded that Worldwide really did mean "other countries", i.e. "none of the above".

Reading what you say it makes sense that "Worldwide" and "Remote" would be synonymous, and therefore you're right to infer that my suggestion would be to rename the entry, rather than create a new one.

I do think "Remote" is preferable, because it's the nomenclature most people are used to in the context of work locations; plus, I don't think it loses the semantic fitness for that list that "Worldwide" has -- as I see it, "remote" is a term defining a location, like "Worldwide" is (in both cases a non-specific one, but the former more precisely meaning "not [necessarily] in the company's physical offices").
